x-pack/plugins/cases/server/client/cases/bulk_update.ts

```diff
@@ -286,6 +288,12 @@ function partitionPatchRequest(
       conflictedCases.push(reqCase);
       // let's try to authorize the conflicted case even though we'll fail after afterwards just in case
       casesToAuthorize.set(foundCase.id, { id: foundCase.id, owner: foundCase.attributes.owner });
+    } else if (
+      foundCase.attributes.status !== reqCase.status &&
```

Comment:
I think we should check for `reqCase.status != null` because the request might have any other field (without the status) and then users with no permission to reopen the case will not be able to update the case even though they do not change the status of the case. Example: `'closed' !== undefined && 'closed' === 'closed' // the expression evaluates to true`.